### PR TITLE
test:  added variable values directly in the example main file

### PR DESCRIPTION
**Pull request by aatreyee2506**
### Description

Instead of exposing schema and topic variables in the examples, hardcoded the data in main.tf of examples.
[Git Issue](https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/issues/331)
### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

Instead of exposing schema and topic variables in the examples ,hardcoded the data in main.tf of examples.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by aatreyee2506** [comment]
cc: @SarikaSinha

**Comment by Ak-sky** [comment]
@aatreyee2506- pre-commit failed, please check
image

File: examples/complete/variables.tf [hunk 1]
```diff
@@ -34,74 +34,74 @@ variable "access_tags" {
...
+#       partitions = 1
+#       config = {
+#         "cleanup.policy"  = "compact,delete"
+#         "retention.ms"    = "86400000"
+#         "retention.bytes" = "1073741824"
+#         "segment.bytes"   = "536870912"
+#       }
+#     }
+#   ]
+# }
```
Comment (Ak-sky) [inline examples/complete/variables.tf:107]: Remove the commented code.

**Review by Ak-sky — DISMISSED** [review]
Please check comments

File: examples/complete/variables.tf (hunk 1 above)
Comment (aatreyee2506) [reply examples/complete/variables.tf]: Removed the commented code.

**Comment by Ak-sky** [comment]
@aatreyee2506, Pre-commit failed again, please always run pre-commit in your local before you push the code.

image

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**